plugins/Makefile.mk

```diff
@@ -23,3 +23,7 @@ include plugins/dmx4linux/Makefile.mk
 include plugins/e131/Makefile.mk
 include plugins/uartdmx/Makefile.mk
 endif
+
+include/ola/plugins/%_plugin_desc.h: plugins/%/README.md plugins/convert_README_to_header.sh plugins/Makefile.mk
```

Comment:
double `%`s don't work unfortunately. Should I rather go for `plugins/%/PluginDescription.h`?